src/view/init.view.ts

```diff
@@ -61,17 +61,17 @@ export class InitView {
     update = false,
   }: ITaskListConfig): Promise<void> {
     if (functionNames?.length) {
-      functionNames.forEach((entry) => {
+      functionNames.forEach((functionName) => {
         this.tasklist.addTask({
-          title: `Initialise ${entry}`,
+          title: `Initialise ${functionName}`,
           task: async () => {
-            this.defaultStructureService.create(entry);
+            this.defaultStructureService.create(functionName);
           },
         });
       });
     } else {
       this.tasklist.addTask({
-        title: `${update ? 'Update files' : 'Initialise example function'}`,
+        title: 'Initializing structure',
```

Comment:
The title should be like the following code. Because in the case we update the bin folder in the user project the text should differ from the init command.
``${update ? 'Update files' : 'Initializing structure'}``